LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License
 
-Copyright (c) Adyen B.V. (https://www.adyen.com)
+Copyright (c) Adyen (https://www.adyen.com)
```

Comment:
Aren't we Adyen NV?

Reply:
We are, but all other SDKs use just "Adyen"